### PR TITLE
1260 type cobol cfg

### PR DESCRIPTION
**Pull request by mayanje**
Builsing TypeCobol/Cobol Control Flow Graph and Report using GraphViz dot file.

### CODE REVIEW

File: TypeCobol.Analysis/TypeCobol.Analysis.csproj [hunk 1]
```diff
@@ -0,0 +1,63 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
```
[the remark this answers is not in the document]
Comment (mayanje) [reply TypeCobol.Analysis/TypeCobol.Analysis.csproj]: Ok, it's done

File: TypeCobol.Analysis.Test/TypeCobol.Analysis.Test.csproj [hunk 2]
```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props')" />
```
[the remark this answers is not in the document]
Comment (mayanje) [reply TypeCobol.Analysis.Test/TypeCobol.Analysis.Test.csproj]: Ok, it's done

File: TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.cs [hunk 3]
```diff
@@ -115,7 +708,139 @@ public override void EndParagraph()
         }
 
         /// <summary>
-        /// Start a sentence
+        /// Link this sentence to the current section or paragraph if any.
+        /// </summary>
+        /// <param name="block">The block to link</param>
+        private void LinkBlockSentenceToCurrentSectionParagraph(BuilderSentence block)
+        {
+            Symbol curSecorPara = ((Symbol)this.CurrentProgramCfgBuilder.CurrentParagraph) ?? this.CurrentProgramCfgBuilder.CurrentSection;
```
[the remark this answers is not in the document]
Comment (mayanje) [reply TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.cs]: Ok, its done

**Comment by smedilol** [comment]
Branch 1260_TypeCobolCFG is now outdated.
See new PR #1735

**Pull request closed without merging**